Detectors/ITSMFT/MFT/workflow/src/TrackFitterSpec.cxx

```diff
@@ -163,13 +170,32 @@ void convertTrack(const T& inTrack, O& outTrack, C& clusters)
     outTrack.createParamAtCluster(tempcluster);
     //std::cout << "Adding cluster " << cls << " to track " << ntrack << " with clusterID " << tempcluster.getSensorID() << " at z = " << tempcluster.getZ() << std::endl;
   }
-
+  outTrack.setMCCompLabels(inTrack.getMCCompLabels(), nClusters);
   //std::cout << "  ** outTrack has getNClusters = " << outTrack.getNClusters() << std::endl;
   //for (auto par = outTrack.rbegin(); par != outTrack.rend(); par++)
   //  std::cout << "     getZ() =  " << par->getZ() << std::endl;
 
   //fit(outTrack, false);
 }
 
+//_________________________________________________________________________________________________
+SMatrix55 TtoSMatrix55(TMatrixD inMatrix)
```

Comment:
@rpezzi will be more efficient to use sym. matrix 
````
using SMatrix55 = ROOT::Math::SMatrix<double, 5,53, ROOT::Math::MatRepSym<double, 5>>;
...
// TMatrix to sym SMatrix
for (auto i = 5; i--;) {
    outMatrix(i, i) = inMatrix(i, i);
    for (auto j = i; j--;) {
        outMatrix(i, j) = inMatrix(i, j);
    }
}

// sym SMatrix to TMatrix
for (auto i = 5; i--;) {
   outMatrix(i, i) = inMatrix(i, i);
    for (auto j = i; j--;) {
        outMatrix(j, i) = outMatrix(i, j) = inMatrix(i, j);
    }
}

````

Reply:
Thanks. Done.